### PR TITLE
interfaces/time-control,netlink-audit: adjust for util-linux compiled with libaudit

### PR DESCRIPTION
**Pull request by jdstrand**
- interfaces/time-control: allow socket NETLINK_AUDIT for hwclock with libaudit
- interfaces/netlink-audit: add missing audit_write capability
- adjust test-snapd-timedate-control-consumer to plugs 'netlink-audit'

### CODE REVIEW

File: interfaces/builtin/netlink_audit_test.go [hunk 1]
```diff
@@ -81,12 +82,20 @@ func (s *NetlinkAuditInterfaceSuite) TestSanitizePlug(c *C) {
 	c.Assert(interfaces.BeforePreparePlug(s.iface, s.plugInfo), IsNil)
 }
 
-func (s *NetlinkAuditInterfaceSuite) TestUsedSecuritySystems(c *C) {
-	seccompSpec := &seccomp.Specification{}
-	err := seccompSpec.AddConnectedPlug(s.iface, s.plug, s.slot)
+func (s *NetlinkAuditInterfaceSuite) TestAppArmorSpec(c *C) {
```
Comment (zyga) [inline interfaces/builtin/netlink_audit_test.go]: Thank you for splitting this per backend, I like it much better this way.

File: interfaces/builtin/time_control.go [hunk 2]
```diff
@@ -95,10 +95,26 @@ capability sys_time,
 
 # As the core snap ships the hwclock utility we can also allow
 # clients to use it now that they have access to the relevant
-# device nodes.
+# device nodes. Note: some invocations of hwclock will try to
+# write to the audit subsystem. We omit 'capability audit_write'
+# and 'capability net_admin' here. Applications requiring
+# logging should plugs 'netlink-audit'.
```
Comment (zyga) [inline interfaces/builtin/time_control.go]: Typo, `should plug`

Comment (jdstrand) [reply interfaces/builtin/time_control.go]: Done

**Comment by zyga** [comment]
```
Formatting wrong in following files:
/tmp/static-unit-tests/src/github.com/snapcore/snapd/interfaces/builtin/netlink_audit_test.go
/tmp/static-unit-tests/src/github.com/snapcore/snapd/interfaces/builtin/time_control_test.go
/tmp/static-unit-tests/src/github.com/snapcore/snapd/interfaces/builtin/netlink_audit_test.go
/tmp/static-unit-tests/src/github.com/snapcore/snapd/interfaces/builtin/time_control_test.go
```

I'll fix that

**Comment by jdstrand** [comment]
@zyga - thanks for the review and formatting fix (/me grumbles about that).

**Pull request merged**